README.md

```diff
@@ -36,7 +36,9 @@ _Feel free to try this out using [our example APK](https://github.com/bamlab/and
 
 1. Install the profiler `yarn add --dev @perf-profiler/e2e`
 
-2. Create a TS script including an e2e performance test,
+2. Branch your android phone, and launch your production app
```

Comment:
We don't need to launch the app. But we can explain that we need to enable [debug mode](https://developer.android.com/studio/debug/dev-options#enable).